plugins.txt

```diff
@@ -32,17 +32,17 @@ View Release Notes=Zeige Veröffentlichungshinweise
 No plugins installed=Keine Plugins installiert
 
 ; New entries (2) - 2021-05-09
-Checking connectivty=
-No response, aborting=
+Checking connectivty=Verbindungen werden geprüft
+No response, aborting=Keine Reaktion, abbrechen
 
 ; New entries (1) - 2021-05-10
-checking=
+checking=prüfe
 
 ; New entries (3) - 2021-05-10
-Delete Plugin=
-Remove Plugin=
-Update Plugin=
+Delete Plugin=Lösche Plugin
+Remove Plugin=Entferne Plugin
+Update Plugin=Aktualisiere Plugin
 
 ; New entries (2) - 2021-05-15
-Plugin File Install Errors=
-Plugin History=
+Plugin File Install Errors=Plugin Dateinstallationsfehler
+Plugin History=Plugin Historie
```

Comment:
@SkyBird1980 Historie gefällt mir eigentlich ganz gut bin mir aber nicht ganzs sicher was würdest du zu "Verlauf" sagen? Weiß nicht in welchem Kontext das steht...

Reply:
Das weis ich auch nicht. Verlauf könnte aber mehr meinen das der Verlauf aller Plugins gemeint ist. Ich lass es erstmal so.